docs/components/Splash.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { useState, useEffect } from "react";
+import styles from "./Splash.module.css";
+
+const Splash = ({ theme }) => {
+  const [isSplashOpen, setSplashOpen] = useState(false);
+  const [display, setDisplay] = useState(false);
+
+  // Effect to only show Splash once the theme has been loaded, so that the light/dark theme logo can display properly underneath
+  useEffect(() => {
+    if (theme) {
+      setSplashOpen(true);
+    }
+  }, [theme]);
+
+  // Effect to set a flag in localstorage if splash screen hasn't been displayed
+  useEffect(() => {
+    if (!localStorage.getItem("splashDisplayed")) {
+      localStorage.setItem("splashDisplayed", "true");
+      setDisplay(true);
+    }
+  }, []);
+
+  const closeSplash = () => {
+    setSplashOpen(false);
+  };
+
+  const stopPropagation = (event) => {
+    event.stopPropagation();
+  };
+
+  const handleSignUp = () => {
+    window.location.href = "https://ethglobal.com/events/autonomous";
+  };
+
+  if (!isSplashOpen) return null;
+
+  return (
+    display && (
+      <div className={styles.splashOverlay} onClick={closeSplash}>
+        <div className={styles.splashContent} onClick={stopPropagation}>
+          <button className={styles.splashClose} onClick={closeSplash}>
+            X
```

Comment:
tiny thing but I find `&times;` to be slightly better for rendering close icons

Reply:
Done, thanks for the suggestion!